apps/vs-code-designer/src/constants.ts

```diff
@@ -157,12 +172,26 @@ export const extInstallTaskName = `${func}: ${extInstallCommand}`;
 export const tasksVersion = '2.0.0';
 export const launchVersion = '0.2.0';
 export const dotnetPublishTaskLabel = 'publish';
+export const dependenciesPathSettingKey = 'dependenciesPath';
+export const defaultDependencyPathValue = path.join(os.homedir(), '.azurelogicapps', 'dependencies');
+export const defaultFuncCoreToolsPathValue = path.join(os.homedir(), '.azurelogicapps', 'dependencies', 'funcCoreTools');
 
 // local.settings.json
 export const localEmulatorConnectionString = 'UseDevelopmentStorage=true';
 
 // host.json
 export const extensionBundleId = 'Microsoft.Azure.Functions.ExtensionBundle.Workflows';
+export const targetBundleKey = 'FUNCTIONS_EXTENSIONBUNDLE_SOURCE_URI';
+export const defaultProductionBundleUrl =
+  'https://cdnforlogicappsv2.blob.core.windows.net/npathakdevex/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle.Workflows/index-v2.json';
+// 'https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle.Workflows/index-v2.json';
```

Comment:
NIT: comment.

Reply:
ah, I need to remove this as well because it has changed and this was used for testing.